cmake/KokkosKernels_Version_Info.cpp.in

```diff
@@ -0,0 +1,31 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#include "KokkosKernels_Version_Info.hpp"
+
+namespace KokkosKernels {
+namespace Impl {
```

Comment:
Why do we need to create this source code file, Kokkos Kernels is typically header only, I would rather we avoid this.